Light.json

```diff
@@ -38840,6 +38849,7 @@
         "type": "Character",
         "subType": "Droid",
         "uniqueness": "*",
+        "destiny": "2 or 5",
```

Comment:
Please use the high number of `5` as the destiny.
For the low number of `2`, please place in a field of `uniqueDestiny: "2`